src/main/java/org/apache/parquet/tools/read/SimpleRecord.java

```diff
@@ -0,0 +1,149 @@
+/* 
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.tools.read;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.*;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Maps;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.node.BinaryNode;
+
+public class SimpleRecord {
+  public static final int TAB_SIZE = 2;
+  protected final List<NameValue> values;
+
+  public SimpleRecord() {
+    this.values = new ArrayList<NameValue>();
+  }
+
+  public void add(String name, Object value) {
+    values.add(new NameValue(name,value));
+  }
+  
+  public List<NameValue> getValues() {
+    return Collections.unmodifiableList(values);
+  }
+
+  public String toString() {
+    return values.toString();
+  }
+
+  public void prettyPrint() {
```

Comment:
May I ask if `prettyPrint` is used (i.e., in production)? Or if it is only used for testing/development? If it is the latter, then perhaps we should not include it in the class.

And also, if we are to remove it, then a lot of the methods/classes can also be removed.